components/automate-chef-io/content/docs/applications-setup.md

```diff
@@ -106,8 +103,8 @@ Chef Automate's TLS certificate file is located at `/hab/svc/automate-load-balan
 
 From Chef Automate:
 
-1. On the command line, run `chef-automate config show`
-1. Copy the `[[global.1.frontend_tls]]` certificate contents from the certificate list
+1. On the command line, run `chef-automate external-cert show`
+1. Copy the certificate contents from the output
 1. Make sure to copy it entirely, including `-----BEGIN CERTIFICATE...END CERTIFICATE-----`
 1. Save the file as `automate.cert` somewhere safe and accessible to you
```

Comment:
Does this need to be `ssl-certificate.crt` if the user is copying it to `/hab/cache/ssl`? See the [certs](https://github.com/habitat-sh/on-prem-builder/blob/master/on-prem-docs/builder-automate.md#step-three-put-the-certs-with-the-install-script) section in the on-prem-docs.

Reply:
The referenced docs are for installing bldr, this procedure is for the hab supervisor. As far as I'm aware, the supervisor will pick up any file with a `.cert` name and use it for SSL connections.

Reply:
The behavior we're relying on here is documented in this section: https://www.habitat.sh/docs/using-builder/#sts=Using%20Custom%20Certificates the part starting with "The Habitat 0.85.0 release in September 2019..." -- which unfortunately doesn't say anything about what the filenames should be. But in any case, if this was working when you and Kyleen first wrote this, it should still be working now.